File: benchmark/comparison/results/blind_py/10.py
def is_prime(n):
    if n < 2:
        return False
    if n == 2:
        return True
    if n % 2 == 0:
        return False
    i = 3
    while i * i <= n:
        if n % i == 0:
            return False
        i += 2
    return True

def fmt(b):
    return "true" if b else "false"

print(fmt(is_prime(2)))
print(fmt(is_prime(3)))
print(fmt(is_prime(4)))
print(fmt(is_prime(17)))
print(fmt(is_prime(1)))
print(fmt(is_prime(0)))

File: benchmark/comparison/results/blind_ts/10.ts
function is_prime(n: number): boolean {
  if (n < 2) return false;
  for (let i = 2; i * i <= n; i++) {
    if (n % i === 0) return false;
  }
  return true;
}

console.log(String(is_prime(2)));
console.log(String(is_prime(3)));
console.log(String(is_prime(4)));
console.log(String(is_prime(17)));
console.log(String(is_prime(1)));
console.log(String(is_prime(0)));
